src/OllamaSharp/MicrosoftAi/Tools/MsAIToolInvoker.cs

```diff
@@ -0,0 +1,65 @@
+using System.Runtime.CompilerServices;
+using OllamaSharp.Models.Chat;
+using AIFunction = Microsoft.Extensions.AI.AIFunction;
+using AIFunctionArguments = Microsoft.Extensions.AI.AIFunctionArguments;
+namespace Triassic.AICore.OllamaAPI;
+
+/// <summary>
+/// A tool invoker that continues a ollamaCLient conversation by sending results from invoked tools back to the ollamaCLient.
+/// </summary>
+public class MsAIToolInvoker()
+{
+	/// <summary>
+	/// Invoke Tool Function
+	/// </summary>
+	/// <param name="toolCalls"></param>
+	/// <param name="chatRequest"></param>
+	/// <param name="cancellationToken"></param>
+	/// <returns></returns>
+	/// <exception cref="Exception"></exception>
+	public static async Task<List<Message>> InvokeAsync(IEnumerable<Message.ToolCall> toolCalls, ChatRequest chatRequest, [EnumeratorCancellation] CancellationToken cancellationToken)
```

Comment:
This method is pretty similar to `ChatContinuingToolInvoker.InvokeAsync()`. Can we get this one reused here?